Form-Controls/index.html

```diff
@@ -13,6 +13,8 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
+        <Label for="first-name">First Name:</Label>
```

Comment:
This file looks unrelated to this PR? Can you work out how to undo this change? Ideally using `git commit --amend` to avoid changing and un-changing this file on this branch?